hydromt_wflow/wflow.py

```diff
@@ -90,19 +76,32 @@ class WflowModel(Model):
 
     def __init__(
         self,
-        root=None,
-        mode="w",
-        config_fn=None,
-        data_libs=None,
-        deltares_data=False,
+        root: Optional[str] = None,
+        mode: Optional[str] = "w",
+        config_fn: Optional[str] = None,
+        data_libs: Union[List, str] = None,
         logger=logger,
+        **artifact_keys,
     ):
+        for lib, version in artifact_keys.items():
+            warnings.warn(
+                "Adding a predefined data catalog as key-word argument is deprecated, "
+                f"add the catalog as '{lib}={version}'"
+                " to the data_libs list instead.",
+                DeprecationWarning,
+                stacklevel=2,
+            )
```

Comment:
Given that you also run `super().__init__()` this might lead to duplicated warnings, I'm not sure, but it would be good to double check

Reply:
Will look at that.